.nvmrc

```diff
@@ -1 +1 @@
-6
```

Comment:
There's a command called "nvm" that allows you to quickly swapped which version of node you are using.

The `.nvmrc` file define which version of node you want to use for the current project. You'll be able to run `nvm use` from within this folder and you will automatically be swapped to node 10. 

Rather than explicitly define which version of node the travis box should use, we sometimes just call `nvm use`